src/components/button/AposButton.vue

```diff
@@ -255,4 +298,10 @@ export default {
     color: var(--a-danger);
   }
 
+  @keyframes animateGradient {
```

Comment:
This animation feels odd. A little dizzying. When I look at it I can't tell what's going on.

Reply:
did implemented this differently, smoother gradation change.